src/data.js

```diff
@@ -1,6 +1,34 @@
 // keep it commonjs export
 // Data is validated against the following schema https://github.com/wesbos/awesome-uses/blob/master/scripts/utils.js#L53-L68
 module.exports = [
+  {
+    name: 'Saurav Khare',
+    description: 'Software Engineer & Frontend Developer',
+    url: 'https://sauravkhare.xyz/uses',
```

Comment:
It looks like this is an empty page right now. Will, you see if you can fix it @SauravKhare?

Reply:
The issue is fixed.